icons/argo-icon.svg

```diff
@@ -0,0 +1 @@
118.526 0 0 1-16.338-35.585c-.985-3.574-2.27-4.363-5.567-3.782-4.459.785-5.83.086-7.777-3.968v-50.744c2.086-4.413 2.98-4.964 7.757-4.781 3.415.13 3.591-.077 4.5-3.536 5.757-21.918 15.726-41.593 31.568-57.98 19.065-19.718 42.673-31.198 69.089-37.424C162 7.382 168.773 6.76 175.526 6Zm115.959 130.12a107.555 107.555 0 0 0-.697-13.642c-.383-3.162-.827-6.327-1.456-9.447-8.69-43.096-34.075-73.001-73.444-91.144-22.583-10.408-46.678-12.154-70.927-7.432-47.243 9.2-78.475 37.555-95.088 82.193-6.346 17.052-7.195 34.975-5.384 53.064a115.874 115.874 0 0 0 10.823 39.172c8.513 17.652 21.008 32.1 36.039 44.48a77.324 77.324 0 0 0 16.29 10.461c3.416 1.604 3.585 1.513 3.571-2.131a59.578 59.578 0 0 0-.146-6.983c-1.815-14.826-2.067-29.736-2.45-44.635a5.706 5.706 0 0 0-2.677-5.126c-5.592-3.892-10.37-8.616-13.398-14.816-6.885-14.099-5.41-34.34 11.828-46.748 1.978-1.424 3.166-2.775 2.873-5.43a25.528 25.528 0 0 1 .463-6.428c2.561-17.26 10.863-31.087 25.027-41.299 9.91-7.145 20.975-11.221 33.247-11.377 7.025-.09 13.862 1.604 20.572 3.555 23.382 6.799 41.385 30.108 41.364 53.765a9.322 9.322 0 0 0 3.89 8.217c6.744 5.385 11.947 12.067 13.719 20.74 3.198 15.655-1.202 28.816-14.235 38.3-3.87 2.816-5.042 5.89-4.849 10.141a23.43 23.43 0 0 1-.022 2.07q-.446 10.462-.902 20.923c-.37 8.517-1.995 16.95-1.698 25.515.111 3.23.085 3.281 2.916 1.857a108.75 108.75 0 0 0 34.802-27.675c19.196-22.945 30.166-49.032 29.949-80.14Zm-166.336 45.874a28.141 28.141 0 1 0 .242-56.28c-15.912-.061-28.409 11.945-28.47 27.353-.065 16.682 11.817 28.86 28.228 28.927Zm84.247-.002c16.018.024 28.144-11.987 28.143-27.876 0-15.877-12.292-28.35-27.908-28.317-15.477.033-28.217 12.588-28.258 27.85-.043 16.06 12.078 28.319 28.023 28.343Zm-21.395 28.161c-3.032 1.858-5.912 4.028-9.16 5.584-7.793 3.734-15.46 3.742-22.973-.706-2.295-1.36-4.472-2.919-6.741-4.324-.681-.421-1.335-1.503-2.19-.905-.636.446-.13 1.444-.01 2.198a72.978 72.978 0 0 1 1.199 7.908c.39 8.217 4.258 14.423 10.802 19.048a14.277 14.277 0 0 0 11.486 2.704c8.226-1.717 14.912-8.932 16.349-17.715.633-3.872.066-7.903 1.59-11.668a1.947 1.947 0 0 0-.352-2.124ZM72.75 138.608c-.183-16.023 2.076-29.888 7.852-43.061 9.199-20.98 23.885-37.128 43.328-49.06a5.507 5.507 0 0 1 6.678.965c.882.882.524 1.718.155 2.601a15.8 15.8 0 0 1-6.983 7.403c-15.693 9.3-26.597 22.847-34.267 39.1a91.664 91.664 0 0 0-9 43.414c.81 19.027 6.394 36.598 17.641 52.17a11.567 11.567 0 0 1 2.518 5.31c.147 1.168.85 2.45-.448 3.316a4.544 4.544 0 0 1-4.195.607 7.304 7.304 0 0 1-3.399-2.696c-10.477-14.229-17.355-29.895-19.351-47.603-.534-4.735-.071-9.486-.53-12.466Zm62.247-98.578a5.448 5.448 0 0 1 5.503-5.563c3.078.016 5.95 3.29 6.044 6.892.052 1.995-4.486 5.669-6.92 5.602-2.738-.076-4.627-2.904-4.627-6.93Zm-6.765 108.777a8.888 8.888 0 0 1 8.54-9.188c5.524-.03 9.75 3.94 9.806 9.21.056 5.168-3.931 9.04-9.293 9.025-5.536-.015-9.026-3.504-9.053-9.047Zm94.481 9.049c-5.559-.033-9.095-3.706-9.034-9.383a9.205 9.205 0 0 1 8.658-8.922 9.551 9.551 0 0 1 9.74 9.353c.008 5.136-4.017 8.983-9.364 8.952Z"/></svg>
```

Comment:
Please check the svg and the change the filename to `argo.svg`
![image](https://user-images.githubusercontent.com/38325136/140574188-46a18c3f-a814-4788-98e4-ebcd31f209fd.png)